joplin/static/js/editor.js

```diff
@@ -0,0 +1,27 @@
+$(function() {
+
+  $('.js-proxy-click').click(function() {
+    let $this = $(this);
+    $this.text($this.data('clicked-text'));
+
+    let $button;
```

Comment:
We'll need to get babel in here if we're going to use fancy js. `let` doesn't work before IE11 and I expect we'll have some editors using shitty versions of IE. https://caniuse.com/#feat=let

Reply:
Ah, forgot about that. I can go oldskool.